### PR TITLE
Deletes a bunch of stuff.

### PR DESCRIPTION
**Pull request by johnnymatthews**

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

File: README.md [hunk 1]
```diff
@@ -101,21 +101,30 @@ To solve this error, perform the following steps:
...
 
-We would **love ❤️ your help** to improve existing items or make new ones even better! [We also have bounties available](https://github.com/ipfs/devgrants/projects/1)!
+3. Run the following command:
+
+   ```shell
+    export NODE_OPTIONS=--openssl-legacy-provider
+   ```
 
-### Issues
+4. Run `npm start`.
```
Comment (TMoMoreau) [inline README.md:117]: Numbers in an order list need to start with "1".

File: README.md [hunk 2]
```diff
@@ -101,21 +101,30 @@ To solve this error, perform the following steps:
    npm start
    ```
 
-You can return to the [Project set-up](#project-set-up) section above and continue with the steps. You can also check [this issue in the Webpack GitHub repository](https://github.com/webpack/webpack/issues/14532) for more information about this error.
+1. Open a terminal.
+2. Navigate into the `ipfs-docs` folder:
 
-## Contribute to documentation
+   ```bash
```
Comment (TMoMoreau) [inline README.md:107]:
````suggestion
   ```shell
````

Try to use `shell` instead of `bash`.

**Review by TMoMoreau — CHANGES_REQUESTED** [review]
Left a comment and a suggestion.

File: README.md (hunk 1 above)
Comment (johnnymatthews) [reply README.md]: Ok nice, changed those.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by TMoMoreau** [comment]
Closing. This was for testing/training.

**Pull request closed without merging**